scripts/index.js

```diff
@@ -0,0 +1,74 @@
+const setIndicatorColor = (tag, value) => {
+    if (value >= 80) {
+        tag.style.color = "red";
+    } else if ((value >= 70) && (value <= 79)) {
+        tag.style.color = "orange";
+    } else if ((value >= 60) && (value <= 69)) {
+        tag.style.color = "yellow";
+    } else if ((value >= 50) && (value <= 59)) {
+        tag.style.color = "green"; 
+    } else {
+        tag.style.color = "teal"
+    };
+};
+
+const setGroundLandscape = (tag, value) => {
```

Comment:
Notice that `setGroundLandscape` and `setIndicatorColor` have a lot of the same code. Consider how you might write a single function that achieves both pieces of functionality.